e2e/runtime_scan_test.go

```diff
@@ -50,6 +51,8 @@ func TestRuntimeScan(t *testing.T) {
 			assert.NilError(t, waitForScanDone(t))
 
 			t.Logf("get runtime scan results...")
+			// wait for refreshing materialized views
+			time.Sleep(common.WaitForMaterializedViewRefreshSecond * time.Second)
```

Comment:
in the tests we could configure the viewRefreshHandler time significantly less than default like 1 second, that way we're not waiting for so long in the tests for things to occur.